aws/ec2.go

```diff
@@ -110,3 +113,394 @@ func nukeAllEc2Instances(session *session.Session, instanceIds []*string) error
+	securityGroups, _ := v.svc.DescribeSecurityGroups(
+		&ec2.DescribeSecurityGroupsInput{
+			Filters: []*ec2.Filter{
+				&ec2.Filter{
+					Name:   awsgo.String("vpc-id"),
+					Values: []*string{awsgo.String(v.VpcId)},
+				},
+			},
+		},
+	)
+	for _, securityGroup := range securityGroups.SecurityGroups {
+		logging.Logger.Infof("...deleting Security Group %s", awsgo.StringValue(securityGroup.GroupId))
+		if *securityGroup.GroupName != "default" {
+			_, err := v.svc.DeleteSecurityGroup(
+				&ec2.DeleteSecurityGroupInput{
+					GroupId: securityGroup.GroupId,
+				},
+			)
+			if err != nil {
+				return errors.WithStackTrace(err)
+			}
+		}
+	}
+	return nil
+}
+
+func (v Vpc) nukeVpc() error {
+	logging.Logger.Infof("...deleting VPC %s", v.VpcId)
+	input := &ec2.DeleteVpcInput{
+		VpcId: awsgo.String(v.VpcId),
+	}
+	_, err := v.svc.DeleteVpc(input)
+	if err != nil {
+		return errors.WithStackTrace(err)
+	}
+	return nil
+}
+
+func (v Vpc) nuke() error {
+	logging.Logger.Infof("Nuking VPC %s in region %s", v.VpcId, v.Region)
+
+	err := v.nukeInternetGateway()
+	if err != nil {
+		logging.Logger.Errorf("Error cleaning up Internet Gateway for VPC %s: %s", v.VpcId, err.Error())
+		return err
+	}
+
+	err = v.nukeSubnets()
+	if err != nil {
+		logging.Logger.Errorf("Error cleaning up Subnets for VPC %s: %s", v.VpcId, err.Error())
+		return err
+	}
+
+	err = v.nukeRouteTables()
+	if err != nil {
+		logging.Logger.Errorf("Error cleaning up Route Tables for VPC %s: %s", v.VpcId, err.Error())
+		return err
+	}
+
+	err = v.nukeNacls()
+	if err != nil {
+		logging.Logger.Errorf("Error cleaning up Network ACLs for VPC %s: %s", v.VpcId, err.Error())
+		return err
+	}
+
+	err = v.nukeSecurityGroups()
+	if err != nil {
+		logging.Logger.Errorf("Error cleaning up Security Groups for VPC %s: %s", v.VpcId, err.Error())
+		return err
+	}
+
+	err = v.nukeVpc()
+	if err != nil {
+		logging.Logger.Infof("Unable to delete VPC %s. Skipping to the next default VPC.", v.VpcId)
+		return err
+	}
+	return nil
+}
+
+func NukeVpcs(vpcs []Vpc) error {
+	for _, vpc := range vpcs {
+		err := vpc.nuke()
+		if err != nil {
+			logging.Logger.Errorf("Skipping to the next default VPC")
+			continue
+		}
+	}
+	logging.Logger.Info("Finished nuking default VPCs in all regions")
+	return nil
+}
+
+type DefaultSecurityGroup struct {
+	GroupName string
+	GroupId   string
+	Region    string
+	svc       ec2iface.EC2API
+}
+
+func DescribeSecurityGroups(svc ec2iface.EC2API) ([]string, error) {
+	var groupIds []string
+	securityGroups, err := svc.DescribeSecurityGroups(&ec2.DescribeSecurityGroupsInput{})
+	if err != nil {
+		return []string{}, errors.WithStackTrace(err)
+	}
+	for _, securityGroup := range securityGroups.SecurityGroups {
+		if *securityGroup.GroupName == "default" {
+			groupIds = append(groupIds, awsgo.StringValue(securityGroup.GroupId))
+		}
+	}
+	return groupIds, nil
+}
+
+func GetSecurityGroups(regions []string) ([]DefaultSecurityGroup, error) {
```

Comment:
```suggestion
func GetDefaultSecurityGroups(regions []string) ([]DefaultSecurityGroup, error) {
```
So it is clear you are only getting the default security groups.